src/style.css

```diff
@@ -0,0 +1,129 @@
+body,
+h1,
+h2,
+h3,
+button {
+  margin: 0;
+  padding: 0;
+  font-size: inherit;
+}
+
+.card {
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  font-style: normal;
+  width: 166px;
+  margin: 0 auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.